### PR TITLE
remove secrets and configmaps from base

### PR DESCRIPTION
**Pull request by george-angel**
- moved to example overlay, using generators

### CODE REVIEW

File: base/local/thanos-rule.yaml [hunk 1]
```diff
@@ -116,23 +105,6 @@ spec:
             memory: 200Mi
           limits:
             memory: 500Mi
-      - name: config-watch
```
Comment (hectorhuertas) [inline base/local/thanos-rule.yaml]: why not keep this as base?

Comment (povilasv) [reply base/local/thanos-rule.yaml]: configmap generator will do the work for this

Comment (povilasv) [reply base/local/thanos-rule.yaml]: > The ConfigMap declared from a ConfigMapGenerator is treated differently. A hash is appended to the name and any change in the ConfigMap will trigger a rolling update.

Comment (hectorhuertas) [reply base/local/thanos-rule.yaml]: ah, I see, thanks!

**Pull request merged**